### DISCUSSION TITLE
Create new (small) contests by myself?

### DISCUSSION THREAD

**Discussion by DE8MSH** [mbridak/not1mm#59] [General]
Hi. Is it possible to enter new small (local) contests by myself? VHF contest 4 eg.?

**Comment by mbridak** [comment]
Hi. Yes it's possible.
But it's complicated.

I think the interface of the program itself would need a little work.
But that's just conmetic.

There's a file called 'Anatomy_Of_A_Plugin.md' which tries to document some of the stuff.

Which contest were you considering?

**Comment by HB9HLI** [comment]
I also have this question because on this side of the Atlantic I'm interested in contests like Helvetia (Swiss contest) and the REF Cup (France). I'm not very good with Python programming  and I don't realize if it's complicated or not, but maybe being able to easily create a contest profile rules myself would help and lighten your workload. :-) I'll have a look Anatomy_Of_A_Plugin.md'. 
73's from Dom HB9HLI.

**Reply by mbridak** [reply]
Hello @HB9HLI I've added Helvetia.

Cheers. K6GTE

**Reply by mbridak** [reply]
@HB9HLI  Hi, I have a question. I was working on REF. But find some documentation discrepancies.

Contest Calendar has this for points and mults:

```text
QSO Points:	French: 6 points per QSO with French station same continent
            French: 15 points per QSO with French station on different continent
            French: 1 point per QSO with non-French station same continent
            French: 2 points per QSO with non-French station on different continent
            non-French: 1 point per QSO with French station same continent
            non-French: 3 points per QSO with French station on different continent

Multipliers:	French/Corsica departments once per band
                French overseas prefixes once per band
                non-French DXCC countries once per band (available only to French stations)
``` 

I went to the REF site and downloaded a rules PDF it says:

```text
Scoring
• QSO Points: each F station count one point from same continent or three points with other continent.
• Multipliers: Sum of departments + DOM/TOM worked per band.
• Final score: QSO points X multipliers = final score
```

These are quite different. The REF site rules are from 2018. So MAYBE they're out of date... Dunno. Was hoping that you had worked it recently and might know what path to follow.

**Comment by mbridak** [comment]
I'll try and add those for you.
I'm sorry the plugin system is far from user friendly.

**Comment by OH9FKJ** [comment]
Hi!
The NRAU SSB competition would also be necessary here in OH country. 
 - Best regards Anssi.

**Comment by HB9HLI** [comment]
Wow, great news, thanks!
I think I will use it next to the Helvetia contest...

73s, Dom.

27 août 2024 22:21:31 Michael Bridak ***@***.***>:

**Comment by JimW3FA** [comment]
Similar question- How hard would it be to add the ARRL 160 contest for this weekend? I see your reply above on plugin difficulty, is there a text template I can try and help with?
Jim

**Comment by mbridak** [comment]
@JimW3FA  arrl 160 is added. did it in a rush, so check it out before the event.

**Comment by TyneDunnock** [comment]
Would it be possible to add a generic contest that will allow the operator to just choose to have serial number plus a text field so that a basic exchange could be logged but without any verification, processing or generation of cabrillo file. 
This weekend was the RSGB Commonwealth Competition and I just needed the serial no to be generated for the log. There was also the Tesla contest (serial no + grid square). I would be happy just to log the details and export to an ADI file that I could process for the contest specific data.

**Comment by mbridak** [comment]
You could just use an existing contest that has a similar exchange.
Here's a [list](https://github.com/mbridak/not1mm/blob/master/contest_exchanges.md) of contests and exchanges.
Like the UKEI DX will work for serial number and grid square.

**Reply by TyneDunnock** [reply]
Thanks, the list is very handy.
I was a bit nervous that I'd be half-way through the contest before encountering some issue related to the genuine contest!
N1MM is pretty well my last link to Windows (not being a gamer or serious user of graphical apps) so, thanks, I'm very pleased that I don't have to maintain a Windows computer any longer!
